platform/pocketbook/koreader.app

```diff
@@ -4,6 +4,10 @@ export LC_ALL="en_US.UTF-8"
 # working directory of koreader
 KOREADER_DIR="/mnt/ext1/applications/koreader"
 
+# load our own shared libraries if possible, solely because we don't control InkView, and we'd rather not it have load duplicate system libs...
+# (We handle this via DT_RPATH for our own stuff).
+export LD_LIBRARY_PATH="${KOREADER_DIR}/libs:${LD_LIBRARY_PATH}"
```

Comment:
```suggestion
export LD_LIBRARY_PATH="${KOREADER_DIR}/libs${LD_LIBRARY_PATH+:}${LD_LIBRARY_PATH}"
```

Reply:
Is that documented somewhere?

That idiom is *very* common as far as I'm aware, and I wasn't aware that a set but empty LD_LIBRARY_PATH meant $PWD (or is that just an empty string after a colon?).

Reply:
Fully empty, no, but with `LD_LIBRARY_PATH=foobar:` the current directory is searched too.

Reply:
Well that's... sneaky and weird.

Let's hope early busybox versions were actually POSIX compliant and won't balk at the `+` parameter expansion ;).